src/main/java/raisetech/student/management/repository/StudentRepository.java

```diff
@@ -21,7 +20,7 @@ public interface StudentRepository {
    *
    * @return 受講生情報の一覧
    */
-  @Select("SELECT * FROM students")
+  @Select("SELECT * FROM students WHERE deleted='false'")
```

Comment:
= の前後には半角スペースほしいですね！
```suggestion
  @Select("SELECT * FROM students WHERE deleted = 'false'")
```

Reply:
修正しました。
18d8e329a85e1ca2badfd916c990be63888f1a35